assert/assertions.go

```diff
@@ -772,8 +772,14 @@ func isEmpty(object interface{}) bool {
 	}
 }
 
-// Empty asserts that the specified object is empty.  I.e. nil, "", false, 0 or either
-// a slice or a channel with len == 0.
+// Empty asserts that the given value is "empty".
+//
+// [Zero values](https://go.dev/ref/spec#The_zero_value) are "Empty".
+// Arrays are "Empty" if every element is the zero value of the type (stricter than "Empty").
```

Comment:
Malformed Zero values link and missing line break between Zero values and Arrays:
<img width="959" alt="image" src="https://github.com/user-attachments/assets/93cd4d06-863e-4c66-a2ed-9e40adfde46b" />